app/upgrades.go

```diff
@@ -21,6 +22,11 @@ func (app *App) RegisterUpgradeHandlers(experimental bool) {
 		// Ref: https://github.com/crypto-org-chain/cronos/issues/755
 		params := app.EvmKeeper.GetParams(ctx)
 		params.ExtraEIPs = []int64{}
+
+		// fix the incorrect value on testnet parameters
```

Comment:
should we reduce the scope only in testnet3?
or because the default London fork enabled is equal to zero in every Cronos chain?

Reply:
yeah, the default value is zero, so it should have no effect on other networks.